README.md

```diff
@@ -0,0 +1,4 @@
+## Flink Cookbook
+
+This repository contains common patterns for (Flink) streaming applications. Each pattern should be runnable and contain a 
+description of the use case. The patterns maybe in Scala or Java.
```

Comment:
Shouldn't need this line break here, since Markdown render's will take care of line wrapping for you. Or were you intentionally trying to fix the width here explicitly?